vault/datadog_checks/vault/vault.py

```diff
@@ -89,6 +89,14 @@ def __init__(self, name, init_config, instances):
         # potential configuration errors as part of the check run phase.
         self.check_initializations.append(self.parse_config)
 
+        self.METRIC_TRANSFORMERS = {
```

Comment:
Make that as a class contant?

Reply:
The value being `self.transform_route_metrics`, it must be defined inside a Vault method/constructor.
Moreover `transform_route_metrics` has to be a Vault method since it's calling `submit_openmetric`, so we can't really make it a class constant

Reply:
Indeed, let's lower-case the name of the variable then